charts/ocis/values.yaml

```diff
@@ -85,20 +85,22 @@ insecure:
   ocisHttpApiInsecure: false
 
 cache:
-  # -- Type of the cache to use. It defaults to no cache or a in memory cache depending on the service.
+  # -- Type of the cache to use. It defaults to no cache.
   # To disable the cache, set to "noop".
-  # Can be set to "redis", "redis-sentinel" or "etcd", then the address of Redis (Sentinel) / etcd node(s) needs to be set to `cache.nodes`.
-  type: ""
+  # Can be set to "redis-sentinel", "etcd" or "nats-js-kv" then the address of Redis (Sentinel) / etcd / nats-js node(s) needs to be set to `cache.nodes`.
+  type: "nats-js-kv"
   # -- Nodes of the cache to use.
-  nodes: []
-  # nodes:
-  #   - redis-master-1.ocis-redis.svc.cluster.local:6379
-  #   - redis-master-2.ocis-redis.svc.cluster.local:6379
+  nodes:
+    - nats.ocis-nats.svc.cluster.local:4222
+  # -- Username to use for authentication
+  authusername: ""
+  # -- Password to use for autentication
+  authpassword: ""
```

Comment:
credentials should only be stored in Secret objects. Please see the `secretRef` section how it has been done for other Secrets

Reply:
I saw that you only put the password into a secret.

From what I know username and password are both typically treated as credential and thus stored in the same secret.

Reply:
Put both in the same secret. Added `authentication` bool flag for configuration